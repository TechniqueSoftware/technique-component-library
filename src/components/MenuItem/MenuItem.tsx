import { default as MuiMenuItem, MenuItemProps as MuiMenuItemProps } from '@material-ui/core/MenuItem';

export type MenuItemProps = MuiMenuItemProps;
export const MenuItem = MuiMenuItem;
