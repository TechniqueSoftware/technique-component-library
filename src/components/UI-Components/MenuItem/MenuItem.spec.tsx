import { shallow } from 'enzyme';
import * as React from 'react';
import { MenuItem } from './MenuItem';

describe('MenuItem', () => {
  test('should render with minimal props', () => {
    const shallowWrapper = shallow(<MenuItem>Foo</MenuItem>);
    expect(shallowWrapper.debug()).toMatchSnapshot();
  });
});
