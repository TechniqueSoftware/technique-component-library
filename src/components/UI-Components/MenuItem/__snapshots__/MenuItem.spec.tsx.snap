// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuItem should render with minimal props 1`] = `
"<ForwardRef(MenuItem) classes={{...}}>
  Foo
</ForwardRef(MenuItem)>"
`;
